test/client/deployStrategies/auraDeploy.test.ts

```diff
@@ -22,7 +22,7 @@ const $$ = testSetup();
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.load('@salesforce/source-deploy-retrieve', 'sdr', ['error_creating_metadata_type']);
 
-describe('Aura Deploy Strategy', () => {
+describe.skip('Aura Deploy Strategy', () => {
```

Comment:
Does this still need to be skipped?

Reply:
these test code that isn't used.  I'd like to skip them until the code is actually removed.